cmd/svcat/broker/deregister_cmd.go

```diff
@@ -25,14 +25,19 @@ import (
 
 // DeregisterCmd contains the info needed to delete a broker
 type DeregisterCmd struct {
+	*command.Namespaced
```

Comment:
Since `Namespaced` has a context on it, and you are embedding the field here, you can now safely remove lines 32 and 38, then change the call from `c.Context.App` to just `c.App`. Here's an example of how that looks with an existing command: 

https://github.com/kubernetes-incubator/service-catalog/blob/97f79ce9088b55cfe153328d171b7239f5464a58/cmd/svcat/instance/get_cmd.go#L91

Reply:
you're not wrong, but I'm also not 100% certain what I'm supposed to use the namespace for to begin with. I think I'm going to put this on hold until I rework it to work with namespaced brokers.